### PR TITLE
Hermez contracts

### PR DESCRIPTION
**Pull request by nitinmittal23**

### CODE REVIEW

File: network/testnet/mumbai/index.json [hunk 1]
```diff
@@ -128,6 +135,18 @@
...
     }
   },
+  "Hermez": {
+    "NetworkName": "Litchi",
+    "ChainId": 1402,
+    "RPC": "https://rpc.public.zkevm-test.net",
+    "Explorer": "https://explorer.public.zkevm-test.net",
+    "SupportsEIP1559": false,
+    "Contracts": {
+      "Bridge": "0x9D98DeAbC42dd696Deb9e40b4f1CAB7dDBF55988",
```
Comment (elias-garcia) [inline network/testnet/mumbai/index.json:145]: This address is the `Bridge proxy` address (same as in the line below). Just pointing this to make sure that this was done intentionally.

File: network/testnet/mumbai/index.json [hunk 2]
```diff
@@ -128,6 +135,18 @@
...
+      "BridgeProxy": "0x9D98DeAbC42dd696Deb9e40b4f1CAB7dDBF55988",
+      "GlobalExitRootManagerL2": "0x762d09Ed110ace4EaC94acbb67b1C35D16C3297b"
```
Comment (elias-garcia) [inline network/testnet/mumbai/index.json:147]: This contract here is from Goerli network while the BridgeProxy on top is from the zkEVM. Do we want to mix things here? This SC address is already declared in L92 of the same file BTW.

File: network/testnet/mumbai/index.json (hunk 1 above)
Comment (nitinmittal23) [reply network/testnet/mumbai/index.json:145]: Actually i am not sure if its a proxy address or not. on Goerli, bridge contract is a proxy contract. but on litchi i am not able to see as the contract is not verified on explorer. if you could provide me with the proxy + implementation address on litchi network or is it not proxy?

File: network/testnet/mumbai/index.json (hunk 2 above)
Comment (nitinmittal23) [reply network/testnet/mumbai/index.json:147]: yes, i will update.

**Comment by elias-garcia** [comment]
BTW, I'll rename `hermez` to `zkevm` everywhere, as it's the official name for the product. Hermez is just how the team is called.

File: network/testnet/mumbai/index.json [hunk 3]
```diff
@@ -128,6 +135,18 @@
       "StateReceiver": "0x0000000000000000000000000000000000001001"
     }
   },
+  "Hermez": {
```
Comment (elias-garcia) [inline network/testnet/mumbai/index.json:138]: I would rename this `Hermez` key here to `zkEVM` as it's the official product name (Hermez is how the team is called only).

File: network/testnet/mumbai/index.json [hunk 4]
```diff
@@ -83,6 +83,13 @@
       "FxERC20RootTunnel": "0x79B77456684AEB0bf7c8Ec50FB9c714d52BdA46c",
       "FxERC721RootTunnel": "0x823eF03B39C339337E451d82cEC57f31316de15F",
       "FxERC1155RootTunnel": "0x186C72e49a3Db03E9161Ca18e52C09DE9468Bfc9"
+    },
+    "HermezContracts": {
```
Comment (elias-garcia) [inline network/testnet/mumbai/index.json:87]: I would rename this to `zkEVMContracts` as well.

**Comment by elias-garcia** [comment]
Looks like the ABIs are outdated, you can pick the new ones from here:

- Bridge: https://github.com/0xPolygonHermez/zkevm-contracts/blob/develop/compiled-contracts/BridgeMock.json
- PoE: https://github.com/0xPolygonHermez/zkevm-contracts/blob/develop/compiled-contracts/ProofOfEfficiencyMock.json
- GlobalExitRootManager: https://github.com/0xPolygonHermez/zkevm-contracts/blob/develop/compiled-contracts/GlobalExitRootManagerMock.json
- GlobalExitRootManagerL2: https://github.com/0xPolygonHermez/zkevm-contracts/blob/develop/compiled-contracts/GlobalExitRootManagerL2Mock.json

File: network/testnet/mumbai/index.json (hunk 3 above)
Comment (nitinmittal23) [reply network/testnet/mumbai/index.json:138]: okay

**Review by rahuldamodar94 — APPROVED** [review]

**Pull request merged**